### DISCUSSION TITLE
What's the best way to contribute to the wiki?

### DISCUSSION THREAD

**Discussion by tech189** [ScoopInstaller/Scoop#4537] [Help and Questions] [answered]
Hi everyone!

The wiki has been very helpful for a beginner trying to make new manifests! But I've noticed a few things such as the use of deprecated examples: e.g. `"re"` in checkver rather than `"regex"`. I would love to make PRs for these kind of things but I couldn't work out how to do that. I found repos that create a separate documentation entirely such as https://github.com/pakeweb/scoop.sh, but I feel that it would be better to edit the central wiki! There's a stackoverflow page about how to edit wikis here: https://stackoverflow.com/questions/10642928/how-to-pull-request-a-wiki-page-on-github but it seems quite convoluted.

So, what is the best way to propose edits to the scoop wiki?

**Comment by rashil2000** [comment, marked as answer]
I think the easiest way right now would be paste a diff/patch here in the discussion itself, and someone with edit access on the wiki will acknowledge and merge it manually.

**Comment by rasa** [comment]
@rashil2000 We could pull the wiki into a repo, to allow us to accept PRs. Then set up a GitHub action to publish it to the wiki (or even https://ScoopInstaller.github.io/ ). See https://stackoverflow.com/a/58692638/1432614

**Reply by rashil2000** [reply]
We could do that, but I don't the Wiki is a source that's active enough to warrant a repo of its own. We are not likely to receive that many contributions there, it's a fairly static place.

**Reply by tech189** [reply]
I think more people are comfortable with sending PRs than diffs so I would prefer the repo option in the long run, but I don't mind sending diffs for now.
